src/events/index.spec.js

```diff
@@ -239,6 +239,113 @@ describe('Events', function() {
     });
   });
 
+  describe('getEventTypesByClient', function() {
+    context('the clientId is provided', function() {
+      let clientId = faker.random.uuid();
+      let promise;
+      let request;
+      let eventTypeFromServer;
+
+      beforeEach(function() {
+        eventTypeFromServer = fixture.build('eventType', { fromServer: true });
+
+        request = {
+          ...baseRequest,
+          get: this.sandbox.stub().resolves(eventTypeFromServer)
+        };
+
+        const events = new Events(baseSdk, request);
+        events._baseUrl = expectedHost;
+
+        promise = events.getEventTypesByClient(clientId);
+      });
+
+      it('gets the eventType from the server', function() {
+        expect(request.get).to.be.calledWith(
+          `${expectedHost}/clients/${clientId}/types`
+        );
+      });
+
+      it('returns the requested event type', function() {
+        return expect(promise).to.be.fulfilled.and.to.eventually.deep.equal(
```

Comment:
Shouldn't need deep here

Reply:
They dont seem to pass w/o `.deep` which is why I added it originally

Reply:
Oh, I see why. You need to stub out `formatPaginatedDataFromServer` and test that it is called